### ISSUE TITLE
Remove All "Free for All" Collator Selection

### ISSUE THREAD

**Issue by joepetrowski** [paritytech/cumulus#2877]
As part of block builder improvements, all parachains should have a configurable collator set (e.g. using Aura). Off the top of my head, this means changing Gluttons, but there may be some others (Penpal, Contracts, Template) that need changing too.

From @rphmeier :

> The "free for all" system does not work for building a backlog of blocks with more than a single collator.

**Comment by bkchr** [comment]
> The "free for all" system does not work for building a backlog of blocks with more than a single collator.

This is right, but these chains should just disable to have unincluded blocks.

**Comment by rphmeier** [comment]
Here's an example of how to continue using free-for-all consensus.

https://github.com/paritytech/cumulus/blob/4a5ec22a3b65629d988bdbeec68e083c67001526/polkadot-parachain/src/service.rs#L924-L974

FWIW we could implement a simple "free-for-all" collator using the new architecture. The only reason those functions are currently considered deprecated is because they imply that it's the _correct_ way to start a collator (and need to be kept in for backcompat reasons).

So this issue could also be "write a new FreeForAll collator using new collator architecture".
